feat: add installation instructions for voice chat feature (#315)
SLNE-Development/community-server-docs: Writerside/topics/general-server/questions/faq.md

   <include from="util.md" element-id="mod-pack"/>

1. **Modrinth installieren**
    - Lade die [Modrinth App](https://modrinth.com/app) herunter
    - Installiere die App
    - Öffne die Modrinth App
2. **Melde dich bei Modrinth mit deinem Microsoft-Konto an**
3. **Modpack installieren**
    - Klicke auf die Suchleiste und suche nach `CastCrafter Survival Server`
    - Klicke auf `Installieren`
    - Starte das Modpack

**Wir wünschen dir viel Spaß auf dem Server!**
</def>

<def title="Wie installiere ich den Sprachchat?" id="install-voicechat">

1. **Modrinth installieren**
    - Lade die [Modrinth App](https://modrinth.com/app) herunter
    - Installiere die App
    - Öffne die Modrinth App
2. **Melde dich bei Modrinth mit deinem Microsoft-Konto an**
3. **Modifikation installieren**
    - Öffne die Modrinth App
    - Erstelle ein neues Profil
      - Loader-Version: Fabric
      - Minecraft Version: [%required_game_version%](%required_game_version_link%)
    - Starte das Profil

> Sobald du den Sprachchat auf dem Netzwerk verwendest, bestätigst du, dass du 

> > die [Regeln für den Sprachchat](rules.md) gelesen hast und akzeptiert.

Reviewer: ```suggestion
> die [Regeln für den Sprachchat](rules.md) gelesen und akzeptiert hast.
```

</tab>
<tab title="GD Launcher" id="minecraft-log-gd-launcher">

Im GD Launcher machst du einen Rechtsklick auf das Modpack-Profil, dann auf `Open Folder`. Dort findest du den Ordner
`logs`.
Darin befindet sich eine Datei namens `latest.log`. Diese solltest du auf einer der oben genannten Seiten hochladen.
</tab>
<tab title="MultiMC" id="minecraft-log-multimc">

In MultiMC klickst du auf `Instanz Bearbeiten`, dann auf `Andere Logs`, wählst oben `logs/latest.log` und klickst dann
auf `Hochladen`.
Du erhältst nun einen Link, den du posten kannst. Alternativ kannst du auch direkt im Tab `Minecraft Log` oben rechts
auf `Hochladen` klicken.
</tab>
<tab title="FTB App" id="minecraft-log-ftb-app">

> In der FTP App klickst du dein Modpack-Profil an, dort oben rechst auf `Settings`, anschließend unten links auf

Reviewer: ```suggestion
In der FTB App klickst du dein Modpack-Profil an, dort oben rechts auf `Settings`, anschließend unten links auf
```

> - Der Log ist nicht der CrashReport.
> - Wenn du den Log als eine lange Chatnachricht sendest, brauchst du keine Hilfe zu erwarten!

> > - Ohne Log, brauchst du keine Hilfe zu erwarten. Wenn du jedoch Probleme hast, den Log zu finden, wird dir gerne

Reviewer: ```suggestion
> - Ohne Log brauchst du keine Hilfe zu erwarten. Wenn du jedoch Probleme hast, den Log zu finden, wird dir gerne
```
